resources/styles/components/task/index.less

```diff
@@ -14,14 +14,18 @@
 .concept-coach-title {
   margin-bottom: 1.5rem;
 
-  .chapter-section-prefix {
+  .chapter-section-prefix, .no-title {
     display: inline-block;
     margin-right: 1rem;
   }
 
-  a {
-    color: @openstax-neutral-light;
+  &:not(.no-title) {
```

Comment:
could this be `.has-title`? That way we don't not need no double negatives? :smile: